pkg/internal/ibmblockcsi/ibmblockcsi.go

```diff
@@ -119,3 +122,86 @@ func (c *IBMBlockCSI) GetCSINodeImage() string {
 	}
 	return c.Spec.Node.Repository + ":" + c.Spec.Node.Tag
 }
+
+func (c *IBMBlockCSI) GetSidecarNames() []string {
+	return []string{
+		config.CSIAttacher,
+		config.CSIProvisioner,
+		config.CSINodeDriverRegistrar,
+		config.LivenessProbe,
+	}
+}
+
+func (c *IBMBlockCSI) GetSidecarByName(name string) *csiv1.CSISidecar {
+	for _, sidecar := range c.Spec.Sidecars {
+		if sidecar.Name == name {
+			return &sidecar
+		}
+	}
+	return nil
+}
+
+func (c *IBMBlockCSI) getSidecarPullPolicyByName(name string) corev1.PullPolicy {
+	sidecar := c.GetSidecarByName(name)
+	if sidecar != nil && sidecar.ImagePullPolicy != "" {
+		return sidecar.ImagePullPolicy
+	}
+	return corev1.PullIfNotPresent
+}
+
+func (c *IBMBlockCSI) GetSidecarImageByName(name string) string {
+	sidecar := c.GetSidecarByName(name)
+	if sidecar != nil {
+		return fmt.Sprintf("%s:%s", sidecar.Repository, sidecar.Tag)
+	}
+	return ""
+}
+
+func (c *IBMBlockCSI) GetDefaultImageByName(platform, name string) string {
```

Comment:
GetDefaultContainerImageByPlatformAndName

Reply:
added platform, but I don't want to add container

Reply:
I don't see the new name. Was it pushed?